.github/workflows/test.yml

```diff
@@ -31,23 +31,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12", "pypy3.10"]
+        python: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12", "3.13", "pypy3.10"]
         os: [ubuntu-latest, windows-latest]
         include:
           - python: "3.7"
-            tox_env: "py37-pytest7"
```

Comment:
Let's drop the eol pythons 

They aren't avaliable on recent builds

Reply:
I think we should stop testing pytest7 also. You ok with that?

Reply:
and testing against pytest 4, 5, 6, 7. seems silly

Reply:
people can pin an old version if they want